src/scripts.js

```diff
@@ -56,28 +58,123 @@ Promise.all([fetchedUserData, fetchedSleepData, fetchedActivityData, fetchedHydr
   })
   .catch(error => console.log(error))
 
-$("#user__activities").on("change", runThisFunction)
+// Promise.all([apiRequest.userInfo, apiRequest.sleepInfo, apiRequest.activityInfo, apiRequest.hydrationInfo])
+//   .then(value => {
+//     userData = value[0];
+//     sleepData = value[1];
+//     activityData = value[2];
+//     hydrationData = value[3];
+//     startApp()
+//   })
+//   .catch(error => console.log(error))
+
+
+$("#user__activities").on("change", displayActivityInput)
+$('.user__input__button').on("click", updateUserInformation)
 
 function startApp() {
   createUserRepo();
   createUserData();
   let userNowId = pickUser();
-  let userNow = getUserById(userNowId, userRepo);
+  currentUser = getUserById(userNowId, userRepo);
+  console.log(getTodaysDate());
   let today = makeToday(userRepo, userNowId, hydrationData);
   //today = 2019/09/22
   let randomHistory = makeRandomDate(userRepo, userNowId, hydrationData);
   displayRandomUserHistory(randomHistory);
-  generateInitialInfo(userNowId, activityRepo, userRepo, today, randomHistory, userNow);
-  generateActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, userNow);
+  generateInitialInfo(userNowId, activityRepo, userRepo, today, randomHistory, currentUser);
+  generateActivityInfo(userNowId, activityRepo, today, userRepo, randomHistory, currentUser);
+}
+
+function updateUserInformation() {
```

Comment:
this is a good place to use switch statement, it fits perfectly. We can consider adjusting default value in the future.